ng-dev/perf/workflow/cli.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright Google LLC
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Argv, CommandModule} from 'yargs';
+import {measureWorkflow} from './workflow.js';
+import {loadWorkflows} from './loader.js';
+import {join} from 'path';
+import {determineRepoBaseDirFromCwd} from '../../utils/repo-directory.js';
+
+interface WorkflowsParams {
+  configFile: string;
+  json: boolean;
+}
+
+/** Builds the checkout pull request command. */
+function builder(yargs: Argv) {
+  return yargs
+    .option('config-file' as 'configFile', {
+      default: '.ng-dev/workflows.yml',
```

Comment:
Maybe something like?

```suggestion
      default: '.ng-dev/dx-perf-workflows.yml',
```